db/migrate/20250902091240_add_fields_to_image_data.rb

```diff
@@ -0,0 +1,8 @@
+class AddFieldsToImageData < ActiveRecord::Migration[8.0]
+  def change
+    change_table :image_data, bulk: true do |t|
+      t.json "dimensions"
+      t.json "crop_data"
```

Comment:
Strictly speaking I'd move this into its own, earlier, commit. (Not entirely sure of the sequencing of db migrate vs pod deployment, if we roll out both the DB changes and the code that references the new fields, in the same PR. Don't we need to ship the DB migration first as a separate PR?)